src/main/java/org/jenkinsci/plugins/scriptsecurity/scripts/ClasspathEntry.java

```diff
@@ -58,10 +58,18 @@ public ClasspathEntry(@Nonnull String path) throws MalformedURLException {
     }
     
     static URL pathToURL(String path) throws MalformedURLException {
+        if (path.isEmpty()) {
+            throw new MalformedURLException("JENKINS-37599: empty classpath entries not allowed");
+        }
         try {
             return new URL(path);
         } catch (MalformedURLException x) {
-            return new File(path).toURI().toURL();
+            File f = new File(path);
+            if (f.isAbsolute()) {
+                return f.toURI().toURL();
+            } else {
+                throw new MalformedURLException("classpath entries must be URLs or absolute file paths");
```

Comment:
@svanoort this was already addressed, look at the revised diff please (not sure what happened to “commented on an ignored diff” in @github)